src/dproto_tcp.erl

```diff
@@ -75,6 +79,13 @@
          Metric :: binary(),
          Time :: pos_integer(),
          Count :: pos_integer()} |
+        {get,
+         Bucket :: binary(),
+         Metric :: binary(),
+         Time :: pos_integer(),
+         Count :: pos_integer(),
+         RR :: rr_opt(),
+         R :: non_neg_integer()} |
```

Comment:
I would like this to be a own type with `default` and `n` as valid options that then decode to `0` and `255` as discussed.

As above passing 'special' numbers feels wrong for an API.